requirements-dev.in

```diff
@@ -1,4 +1,3 @@
-ansible
```

Comment:
This should be replaced by `ansible-core`, so we can run `ansible-galaxy` in the `Jenkinsfile`.